### PR TITLE
MGVDU-214: Activities Component

### PR DESCRIPTION
**Pull request by jasonstanbery**
I assume there are going to be changes, so hit me with them.

### CODE REVIEW

**Comment by jasonstanbery** [comment]
Totally forgot things like alt tag. Going to update this and reopen it later today.

File: styleguide/source/_patterns/05-pages/ACTION-get-a-state-park-pass.json [hunk 1]
```diff
@@ -501,6 +501,45 @@
...
+      "data": {
+        "activities": [{
+          "image": "http://placehold.it/190x107",
+          "title": "Hiking & Walking",
+          "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi eu tellus sit amet turpis convallis finibus at id nulla.",
+          "linkTitle": "Details & Maps",
+          "href": "#"
+        },{
+          "image": "http://placehold.it/190x107",
+          "title": "Camping",
```
Comment (gender404) [inline styleguide/source/_patterns/05-pages/ACTION-get-a-state-park-pass.json:517]: image size seems small.  Are there any browser widths were the image is being stretched beyond these dimensions?

File: styleguide/source/assets/scss/02-molecules/_action-activities.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  &__description {
+
+  }
+
+  &__link {
+    @include ma-chevron;
+    @include ma-link-underline;
+  }
+
+  &:last-child {
```
Comment (gender404) [inline styleguide/source/assets/scss/02-molecules/_action-activities.scss:59]: we should probably wrap this component in a blank div just to make sure :last-child always works.  outside of the for loop

Comment (gender404) [reply styleguide/source/assets/scss/02-molecules/_action-activities.scss:59]: You could also use & + & { border-top: 1px solid; } which skips the first item so no override border:none needed on last child

**Review by gender404 — COMMENTED** [review]
only a few items.  Nothing too major

File: styleguide/source/_patterns/05-pages/ACTION-get-a-state-park-pass.json (hunk 1 above)
Comment (jasonstanbery) [reply styleguide/source/_patterns/05-pages/ACTION-get-a-state-park-pass.json:517]: I'm not scaling the image up, but maybe I should (or talk to Serena). It's hard to tell what her intentions were.

**Pull request closed without merging**